Source/WebCore/platform/ios/VideoPresentationInterfaceIOS.mm

```diff
@@ -136,13 +140,77 @@ @interface UIViewController ()
     videoDimensionsChanged(model ? model->videoDimensions() : FloatSize());
 }
 
+void VideoPresentationInterfaceIOS::ensurePipPlacardIsShowing()
+{
+    if (m_pipPlacard) {
+        [m_pipPlacard setHidden:NO];
+        return;
+    }
+    auto pipPlacard = adoptNS([PAL::allocUIViewInstance() initWithFrame:[layerHostView() bounds]]);
+    pipPlacard.get().backgroundColor = (UIColor *)[PAL::getUIColorClass() blackColor];
+    pipPlacard.get().translatesAutoresizingMaskIntoConstraints = NO;
+
+    auto imageColor = (UIColor *)[PAL::getUIColorClass() colorWithRed:164.0 / 255.0 green:164.0 / 255.0 blue:164.0 / 255.0 alpha:1];
+    auto image = [[[PAL::getUIImageClass() systemImageNamed:@"pip"] imageWithTintColor:imageColor renderingMode:UIImageRenderingModeAlwaysOriginal] imageWithConfiguration:[PAL::getUIImageSymbolConfigurationClass() configurationWithWeight:UIImageSymbolWeightThin]];
+
+    auto imageView = adoptNS([PAL::allocUIImageViewInstance() initWithImage:image]);
+    imageView.get().contentMode = UIViewContentModeScaleAspectFit;
+    imageView.get().translatesAutoresizingMaskIntoConstraints = NO;
+
+    [pipPlacard.get() addSubview:imageView.get() ];
+
+    auto pipLabel = adoptNS([PAL::allocUILabelInstance() init]);
+    pipLabel.get().text = @"This video is playing in picture in picture.";
+    pipLabel.get().textAlignment = NSTextAlignmentCenter;
+    pipLabel.get().textColor = imageColor;
+    pipLabel.get().font = [PAL::getUIFontClass() systemFontOfSize:16];
```

Comment:
Do you really want a fixed font size, or should this use dynamic type / text styles?

Reply:
I based this decision off of placard.css, where the font size of the text is set to a constant of 13px (I thought 16px looked a bit better)

Reply:
You might want to use `UIFontTextStyleCallout` (which is 16 [by default](https://developer.apple.com/design/human-interface-guidelines/typography)), so that the text adapts for users that have [custom accessibility settings](https://support.apple.com/en-us/102453). 

https://developer.apple.com/documentation/uikit/uifont/textstyle/callout?language=objc